models.py

```diff
@@ -6,12 +6,13 @@
     CascadeBehaviour,
     NotificationState,
     Tablenames,
-    Notification,
+    Notification as NotificationEnums,
```

Comment:
why was this renamed?

Reply:
Because before we had Notification imported and Notification as a class defined in the same file

Reply:
makes sense, can be resolved